src/chrome/content/rules/AJC.com.xml

```diff
@@ -1,27 +1,17 @@
 <!--
-	^ajc.com: Handshake fails
-	www.ajc.com: 504, Akamai
-
-
-	Problematic hosts in *ajc.com:
-
-		- *.blog *
-
-	* Mismatched
+	Other Atlanta Journal Constitution rulesets:
+		* MyAJC.com.xml
 
+	Numerous unsupported subdomains (>300 found by Sublist3r)
```

Comment:
I just re-ran Sublist3r and it found on 186 domains.  52 of those domains are of the form *.blog.ajc.com (50 of those seem to be active, 2 for testing purposes). Another probably 80 or so are domains that aren't meant to be directly accessed (ex: peer1-xen5.ajc.com ). Then they've got the rest which are subdomains like internships.ajc.com or teamplayer.ajc.com . So besides the blogs, there isn't much regularity.

Here's a link to that output from Sublist3r in case you're interested: https://pastebin.com/XL24jkSW